.travis.yml

```diff
@@ -14,6 +14,9 @@ env:
     - COMPONENT=scheduler                 TEST_ARGS='--worker-metrics --worker-send --beat'
     - COMPONENT=stage-based-messaging     TEST_ARGS='--worker-metrics --worker-send'
     - COMPONENT=auth-api                  TEST_ARGS='--no-worker'
+    - COMPONENT=control-interface         TEST_ARGS='--no-worker'                           DATABASE_ENV_NAME=SEED_CONTROL_INTERFACE_DATABASE LOGIN_URL=login LOGIN_TITLE="Seed Control Interface &middot; Login"
+    - COMPONENT=identity-store            TEST_ARGS=''                                      DATABASE_ENV_NAME=IDENTITIES_DATABASE CELERY_WORKER_QUEUE=mediumpriority,priority,metrics,seed_identity_store
+    - COMPONENT=service-rating            TEST_ARGS=''                                      CELERY_WORKER_QUEUE=seed_service_rating,priority,mediumpriority,metrics,celery
```

Comment:
One thing you can do, if it makes your life easier is change:
```yaml
env:
  matrix:
    - COMPONENT=foo TEST_ARGS='bar'
    - COMPONENT=baz TEST_ARGS='foobar'
```
...to:
```yaml
matrix:
  include:
    - env:
      - COMPONENT=foo
      - TEST_ARGS='bar'
    - env:
      - COMPONENT=baz
      - TEST_ARGS='foobar'
```
...but it's up to you whether you want to do that.